test/mutable/entries_keys_values.js

```diff
@@ -79,13 +79,10 @@ describe('Mutable Data Entries', () => {
       .then((keys) => should(keys.length).equal(Object.keys(TEST_ENTRIES).length))
   );
 
-  it('check list of keys', (done) => {
-    app.mutableData.newRandomPublic(TYPE_TAG)
+  it('check list of keys', () => app.mutableData.newRandomPublic(TYPE_TAG)
       .then((m) => m.quickSetup(TEST_ENTRIES).then(() => m.getKeys()))
       .then((keys) => Promise.all(keys.map((key) =>
-        should(TEST_ENTRIES).have.ownProperty(key.toString())
-      )).then(() => done(), (err) => done(err)));
```

Comment:
This will give you false positives

Reply:
I'm able to fail this test if I give an incorrect value to `should(TEST_ENTRIES).have.ownProperty(...`